src/view/eventsList.js

```diff
@@ -1,3 +1,24 @@
-export const createEventListTemplate = () => (
+import { createElement } from '../helpers/render';
+const createEventListTemplate = () => (
```

Comment:
```suggestion
import { createElement } from '../helpers/render';

const createEventListTemplate = () => (
```

логические куски кода разделяются пустыми строками